packages/frontend/src/translations/locales/en/translation.json

```diff
@@ -1535,6 +1535,7 @@
         "function": "Function:",
         "gasLimit": "Gas Limit",
         "hereAreSomeDetails": "Here are some details that will help you.",
+        "addFullAccessKey": "Add Full Access Key detected:It appears that an external app is trying to add a full access key to your account in this transaction. This would allow them to drain your account. We are preventing the execution of this transaction.",
```

Comment:
```suggestion
        "addFullAccessKey": "An external application is trying to add a full access key to your account. This would allow them to drain your account. For security reasons, we are blocking the execution of this transaction.",
```